app/routes/groups.tsx

```diff
@@ -1,15 +1,26 @@
 import { Card } from '~/components/ui/containers';
 import { useGroups } from '~/hooks/useGroups';
+import { Link } from '@remix-run/react';
+import { LinkButton } from '~/components/ui/forms';
 
 export default function Group() {
   const groups = useGroups();
   return (
-    <Card>
-      <ul className="max-w-md space-y-1 text-gray-500 list-disc list-inside dark:text-gray-400">
-        {groups?.map((group) => {
-          return <li key={group.id}>{group.name}</li>;
-        })}
-      </ul>
-    </Card>
+    <>
+      <Card>
+        <ul className="max-w-md space-y-1 text-gray-500 list-disc list-inside dark:text-gray-400">
+          {groups?.map((group) => {
+            return (
+              <li key={group.id}>
+                <Link to={group.id}>{group.name}</Link>
```

Comment:
Personal preference: Maybe let's use absolute paths only. So if we ever move this code into a reusable component or into a different route, it still works!